lib/samples/change_viewpoint/change_viewpoint.dart

```diff
@@ -0,0 +1,191 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import 'package:arcgis_maps/arcgis_maps.dart';
+import 'package:flutter/material.dart';
+import '../../utils/sample_state_support.dart';
+
+class ChangeViewpoint extends StatefulWidget {
+  const ChangeViewpoint({super.key});
+
+  @override
+  State<ChangeViewpoint> createState() => _ChangeViewpointState();
+}
+
+class _ChangeViewpointState extends State<ChangeViewpoint>
+    with SampleStateSupport {
+  // Create a controller for the map view.
+  final _mapViewController = ArcGISMapView.createController();
+
+  // Define the Redlands polygon.
+  late PolygonBuilder _redlandsEnvelope;
+
+  // Define the Edinburgh polygon.
+  late PolygonBuilder _edinburghEnvelope;
+
+  // String array to store titles for the viewpoints specified above.
+  final _viewpointTitles = [
```

Comment:
Could you use `enum` to define the testing viewpoint types?

Reply:
Yes, but wouldn't it look complex from a user point of view ? Since we want to keep the sample simple, I'm wondering if this wouldn't make complex ?

Reply:
The samples are not consistent on this -- manage_features.dart uses String for the damageType dropdown.

Reply:
Actually manage_features.dart uses both approaches -- damageType is very simple and uses String, and featureManagementOptions uses a custom enum with a `getLabel()` method.

Reply:
As a suggestion, we could use a String here for now, and open a Discussion on whether we should always use a custom enum for dropdown menus.